Add `title` and `titleAlignment` options (#59)
sindresorhus/boxen: readme.md

   ╚═════════════╝

*/

> console.log(boxen('unicorns love rainbows', {title: 'rainbow', titleAlign: 'center'}));

Reviewer: ```suggestion
console.log(boxen('unicorns love rainbows', {title: 'magical', titleAlign: 'center'}));
```

Nitpick, but looks better than repeating a word.

Reduce opacity of the border.

##### title

Type: `string`

> Add a title to the box. If needed the box wil enlarge to fit the title.

Reviewer: ```suggestion
Add a title to the box.

If needed, the box will horizontally expand to fit the title.
```

> Add a title to the box. If needed, the box will horizontally expand to fit the title.

Reviewer: readme and index.d.ts should be in sync.

Example:
```js
console.log(boxen('foo bar', {title: 'example'}));
/*
┌ example ┐
│foo bar  │
└─────────┘
*/
```

##### titleAlign

Type: `string`\

> Default: `left`

Reviewer: ```suggestion
Default: `'left'`
```

Align the title in the top bar.

Values:
- `'left'`
```js
/*
┌ example ──────┐
│foo bar foo bar│
└───────────────┘
*/
```
- `'center'`
```js
/*
┌─── example ───┐
│foo bar foo bar│
└───────────────┘
*/
```
- `'right'`
```js
/*
┌────── example ┐
│foo bar foo bar│
└───────────────┘
*/

> ```

Reviewer: These examples are missing in index.d.ts
